pyro/nn/auto_reg_nn.py

```diff
@@ -121,22 +123,34 @@ class AutoRegressiveNN(nn.Module):
         autoregressive factorization. in particular for the identity permutation the autoregressive structure
         is such that the Jacobian is upper triangular. By default this is chosen at random.
     :type permutation: torch.LongTensor
+    :param skip_connections: Whether to add skip connections from the input to the output.
+    :type skip_connections: bool
+    :param nonlinearity: The nonlinearity to use in the feedforward network.
```

Comment:
give an exmaple here? `nn....`?

Reply:
I've put 3 examples now that should clear up what the output is of arn(x)

Reply:
sorry i meant for the param `nonlinearity`

Reply:
Ah okay!